docs/conf.py

```diff
@@ -225,6 +225,175 @@
 
 autodoc_member_order = 'groupwise'
 
+# -- CLI documentation  -------------------------------------------------------
+"""Improves the CLI documentation section.
+
+In order to have separate html pages for each module inside library\\cli
+additional documentation files(.rst) need to be generated in docs\\cli folder.
+
+Note:
+    This process assumes that each cli module represent a group of subcommands.
+"""
+
+# Library-command hash table. Created to address sub-command group names that
+# differ from their library name. Format: {library_name : command_name}
+ht_lib_command = {}
+
+# Repository-library hash table. Creared to address library names that
+#  differ from their repository name (beyond the dash-lower dash difference).
+# Format: {repo_name : library_name}
+ht_repo_lib = {}
+
+
+def create_CLI_files():
```

Comment:
`cli` should be lower case to conform with PEP8.

Reply:
Done!